sklearn/metrics/cluster/supervised.py

```diff
@@ -734,6 +765,12 @@ def normalized_mutual_info_score(labels_true, labels_pred):
     labels_pred : array, shape = [n_samples]
         A clustering of the data into disjoint subsets.
 
+    average_method : string or None, optional (default: None)
+        How to compute the normalizer in the denominator. Possible options
+        are 'min', 'sqrt', 'sum', and 'max'.
+        If None, 'sqrt' will be used, matching the behavior of
+        `v_measure_score`. 
```

Comment:
single backticks do nothing.

Reply:
Add ``..versionadded``

Reply:
Unsure what `..versionadded` is.

Reply:
http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionadded

do ``git grep versionadded`` to see how we use it.

Reply:
Ah, so which version would that be?

Reply:
0.20